airrohr-firmware/mhz19.cpp

```diff
@@ -0,0 +1,150 @@
+ /***********************************************************************
+ *    MH-Z19 CO2 sensor driver                                          *
+ *    Copyright (C) 2017-2019 Michael Dreher <michael (at) 5dot1.de>    *
+ *                                                                      *
+ * This program is free software: you can redistribute it and/or modify *
+ * it under the terms of the GNU General Public License as published by *
+ * the Free Software Foundation, either version 3 of the License, or    *
+ * (at your option) any later version.                                  *
+ *                                                                      *
+ * This program is distributed in the hope that it will be useful,      *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of       *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the        *
+ * GNU General Public License for more details.                         *
+ *                                                                      *
+ * You should have received a copy of the GNU General Public License    *
+ * along with this program. If not, see <http://www.gnu.org/licenses/>. *
+ ***********************************************************************/
+
+#include "mhz19.h"
+
+CMHZ19Sensor::CMHZ19Sensor(Stream& serialPort) :
+	m_pSerialPort(&serialPort),
+	ppmMaxValue(0xffff)
+{
+}
+
+/*
+set the measuring range to 1000, 2000, 3000 or 5000
+ppm range for sensor MH-Z19 is set to 2000 when sensor is delivered
+*/
+bool CMHZ19Sensor::SetRange(uint16_t range)
```

Comment:
Borderline nit-pick here: I see most of the rest of the code to use functions with the lower camelcase schema, aka this could be setRange() to be more consistent

Reply:
now using camelCasing